_includes/search.html

```diff
@@ -0,0 +1,36 @@
+<div class="row">
+    <div class="col-md-8 col-md-offset-2 searchrow text-center">
+        <div class="col-sm-11 no-padding">
+            <div class="input-group">
+                <div class="input-group-btn" >
+                    <div class="btn-group" role="group"> 
+                        <button type="button" class="btn btn-primary" >
+                            <span class="glyphicon glyphicon-search" aria-hidden="true"></span>
+                        </button>
+                    </div>
+                </div><input id="search-box" type="text" class="form-control guides-input-box" placeholder="Search guides....">
+            </div>
+        </div>
+        <div class="col-sm-1 no-padding"> 
+            <button class="btn tags-filter" type="button" data-toggle="collapse" data-target="#filter-options" aria-expanded="false" aria-controls="filter-options">
+                Filter <i class="fa fa-caret-down" aria-hidden="true"></i>
+            </button>  
+        </div>
+    </div>
+    <div class="col-md-8 col-md-offset-2 filter-options">
+        <div class="collapse" id="filter-options" aria-expanded="true">
+            <div class="card card-body">
+                <p>Tags</p>
+                <div class="tags"></div>
+            </div>
+        </div>
+    </div>
+    <div class="text-center">
+            <div class="cloud-filter">
+                <button id="aws" value="aws" ga-event-category="guide-cloud-filter" ga-event-action="filter-aws"><img  src="/assets/img/logos-technologies/aws-logo.png" alt="aws logo"></button>
+                <button id="gcp" value="gcp" ga-event-category="guide-cloud-filter" ga-event-action="filter-gcp"><img src="/assets/img/logos-technologies/gcp-logo.png" alt="aws logo"></button>
+                <button id="azure" value="azure" ga-event-category="guide-cloud-filter" ga-event-action="filter-azure"><img src="/assets/img/logos-technologies/azure-logo.png" alt="aws logo"></button>
+            </div>
```

Comment:
The image quality of these logos doesnt look really good
<img width="351" alt="Screen Shot 2019-09-05 at 14 07 56" src="https://user-images.githubusercontent.com/1516418/64352018-37b32280-cff3-11e9-99f0-04989975c394.png">

Reply:
Also it still needs to be marked as active after it's clicked (and not just on hover)
<img width="370" alt="Screen Shot 2019-09-05 at 14 13 02" src="https://user-images.githubusercontent.com/1516418/64352481-06872200-cff4-11e9-84bc-fa7ca93b0155.png">

Reply:
It marks as on hover, focus and active.

Reply:
I dont think the active one is working, it disappears right after you click it